examples/server_sync.py

```diff
@@ -180,7 +180,7 @@ def run_server():
             port=port,  # serial port
             custom_functions=[],  # allow custom handling
             framer=framer,  # The framer strategy to use
-            handler=None,  # handler for each session
+            # handler=None,  # handler for each session
```

Comment:
Where did you see problems with this code ? I run the server daily and have so far not had any problems.

Reply:
```
/Users/sanjay/.virtualenvs/pymodbus/bin/python /Users/sanjay/repo/pymodbus/examples/server_sync.py --log debug --comm serial --framer rtu --port /tmp/ptyp0
22:32:23 INFO  server_sync:67 ### Create datastore
22:32:23 INFO  server_sync:142 ### start server
Traceback (most recent call last):
  File "/Users/sanjay/repo/pymodbus/examples/server_sync.py", line 295, in <module>
    run_server()
  File "/Users/sanjay/repo/pymodbus/examples/server_sync.py", line 176, in run_server
    StartSerialServer(
  File "/Users/sanjay/repo/pymodbus/pymodbus/server/sync.py", line 750, in StartSerialServer
    server = ModbusSerialServer(context, framer, identity=identity, **kwargs)
  File "/Users/sanjay/repo/pymodbus/pymodbus/server/sync.py", line 583, in __init__
    self._build_handler(kwargs.get("handler", CustomSingleRequestHandler))
  File "/Users/sanjay/repo/pymodbus/pymodbus/server/sync.py", line 613, in _build_handler
    self.handler = handler(request, (self.device, self.device), self)
TypeError: 'NoneType' object is not callable
```

Reply:
then the correct way would be to solve it in sync.py

Reply:
As I said in the earlier message, I do not have a clear idea of what are the kind of changes you are planning to bring in.  I am trying to avoid the client/server code base as much as possible and trying to work around the examples.